src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = days * 40;
+  let totalCost = price;
+
+  if (days >= 7) {
+    totalCost = price - 50;
```

Comment:
Instead of reassigning `totalCost`, you could directly calculate the discounted price here: `let totalCost = price - 50;` This avoids the need to initialize `totalCost` with `price` earlier.